Makefile

```diff
@@ -28,7 +28,7 @@ VERSION               := $(GIT_TAG)
 endif
 
 # should we build the static files?
-ifneq (,$(filter $@,codegen|lint|test|docs))
+ifneq (,$(filter $(MAKECMDGOALS),codegen lint test docs start))
```

Comment:
Only GNU Make supports `$@`. `$(MAKECMDGOALS)` is supported by all makes.

Also, `make start` does not need static files